lib/functions-api.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+function current_event_call($data) {
+  $current_event_id = IGV_get_option('_igv_site_options', '_igv_radio_evento_current');
+
+  // Check if no event is selected
+  if (empty($current_event_id)) {
```

Comment:
Is this not a really bad pattern for this? Everytime a show becomes live they have to manually set that show to live, even tho we have the data for the show as a JS object already from the Program feature?